controller/location.go

```diff
@@ -0,0 +1,69 @@
+package controller
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	"github.com/marcodeltorob/golang-bootcamp-2020/model"
+	"github.com/unrolled/render"
```

Comment:
By convention, imported packages in a file should be separated into three separated blocks:

1st block: Golang library packages
2nd block: Your project's own dependencies (local packages)
3rd block: 3rd party packages

```suggestion
	"net/http"
	"strconv"
	
	"github.com/marcodeltorob/golang-bootcamp-2020/model"

	"github.com/gorilla/mux"
	"github.com/unrolled/render"
```